css/styles.css

```diff
@@ -665,6 +665,11 @@ form textarea {
     margin-bottom: 3em;
   }
 
+  .language-name p {
+    margin-top: 1em;
+    margin-left: 0;
+  }
+
   #portfolio p {
     font-weight: 500;
   }
```

Comment:
You should remove this line of code from your codebase since it's not useful. Leaving unused code in your codebase makes it harder to read and reason about